tests/unit/extensions/test_kde_neon_6.py

```diff
@@ -89,7 +134,27 @@ def test_is_experimental():
 
 def test_get_app_snippet(kde_neon_6_extension):
     assert kde_neon_6_extension.get_app_snippet() == {
-        "command-chain": ["snap/command-chain/desktop-launch6"],
+        "command-chain": ["snap/command-chain/desktop-launch"],
+        "plugs": [
+            "desktop",
+            "desktop-legacy",
+            "opengl",
+            "wayland",
+            "x11",
+            "audio-playback",
+            "unity7",
+            "network",
```

Comment:
Please remove this, all the snaps that need are give network plug already. Not all snaps should be given network access. I have checked with most of the KDE snaps.

Reply:
I am not going to break 200 snaps because a handful don't use network. I already told you this.

Reply:
Snaps which needs it already have it and there are not many snaps who even use network, and they have those explicitly added. @lengau @mr-cal WDYT?